sympy/polys/specialpolys.py

```diff
@@ -108,10 +108,7 @@ def random_poly(x, n, inf, sup, domain=ZZ, polys=False):
     """Return a polynomial of degree ``n`` with coefficients in ``[inf, sup]``. """
```

Comment:
Looks like you missed this docstring

Reply:
Also, the docstring of symmetric_poly above (line 90) should have the same phrase about polys flag because it also works the same way. We can't get rid of **args there, but they should be documented.  Also, per #14030 a comment like "# TODO: use an explicit keyword argument when Python 2 support is dropped" should be added to `symmetric_poly`.